### ISSUE TITLE
[Documentation]: Workflows visualization

### ISSUE THREAD

**Issue by kun432** [run-llama/llama_index#15133]
### Documentation Issue Description

should be `draw_all_possible_flows` instead of `draw_all_possible_paths`?

https://github.com/run-llama/llama_index/blob/fcc764a22052eb1bb3bc66caca10214906b63816/docs/docs/module_guides/workflow/index.md?plain=1#L152-L163

### Documentation Link

https://docs.llamaindex.ai/en/latest/module_guides/workflow/#drawing-the-workflow

**Comment by kun432** [comment]
@dosu

don't think so. `draw_all_possible_paths` returns ImportError.

```
ImportError: cannot import name 'draw_all_possible_paths' from 'llama_index.core.workflow' (/usr/local/lib/python3.10/dist-packages/llama_index/core/workflow/__init__.py)
```

you should check again below.

https://github.com/run-llama/llama_index/blob/fcc764a22052eb1bb3bc66caca10214906b63816/llama-index-core/llama_index/core/workflow/__init__.py#L16-L29

**Comment by kun432** [comment]
fixed in #15154 

closed
